### PR TITLE
cert-manager: fast-forward to upstream 1e606b3e

### PR DESCRIPTION
**Pull request by munnerz**
* Bump chart and manifests for v0.3.0 release (jetstack/cert-manager#575)
* add pod labels (jetstack/cert-manager#563)
* Add leaderElection.namespace option to helm chart (jetstack/cert-manager#548)
* Add clusterResourceNamespace option to Helm chart (jetstack/cert-manager#547)
* Fix ingress-shim cli flags (jetstack/cert-manager#526)
* Create chart OWNERS (jetstack/cert-manager#528)
* Link ingress-shim into main controller binary (jetstack/cert-manager#502)
* readthedocs documentation rewrite (jetstack/cert-manager#428)
* add selector, as otherwise it fails on GKE k8s v1.9.x (jetstack/cert-manager#440)
* Add podAnnotations (jetstack/cert-manager#387)
* Document common ingressShim.extraArgs use case in chart (jetstack/cert-manager#382)
* Helm Chart: Add support for affinity and tolerations (jetstack/cert-manager#350)
* Set default cluster resource namespace to current pod namespace (jetstack/cert-manager#329)
* Create a Namespace resource as part of the static manifest bundle (jetstack/cert-manager#340)
* Add default shortNames to certificates CRD (jetstack/cert-manager#312)
* fix: Use ConfigMaps for leaderelection (jetstack/cert-manager#327)

/cc @kragniz @simonswine

### CODE REVIEW

File: stable/cert-manager/templates/NOTES.txt [hunk 1]
```diff
@@ -6,10 +6,10 @@ or Issuer resource (for example, by creating a 'letsencrypt-staging' issuer).
...
 
-https://cert-manager.readthedocs.io/en/latest/reference/issuers.html
+http://cert-manager.readthedocs.io/en/latest/reference/issuers.html
 
 For information on how to configure cert-manager to automatically provision
 Certificates for Ingress resources, take a look at the `ingress-shim`
 documentation:
 
-https://cert-manager.readthedocs.io/en/latest/reference/ingress-shim.html
+http://cert-manager.readthedocs.io/en/latest/reference/ingress-shim.html
```
Comment (munnerz) [inline stable/cert-manager/templates/NOTES.txt:15]: 😬 just spotted this!

I'll make a follow up PR upstream to revert this back.

**Comment by munnerz** [comment]
This e2e check is new 😄

Image tag was not pushed previously - it should now be released.

/retest

**Comment by PierluigiLenociAkelius** [comment]
[Please](https://memegenerator.net/img/instances/71382158/please-approve.jpg)

**Review by kragniz — APPROVED** [review]
/lgtm

File: stable/cert-manager/templates/00-namespace.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{{ if .Values.createNamespaceResource }}
```
Comment (unguiculus) [inline stable/cert-manager/templates/00-namespace.yaml:1]: What's the point of this? Helm creates namespaces automatically, so this has to fail.

File: stable/cert-manager/templates/NOTES.txt (hunk 1 above)
Comment (unguiculus) [reply stable/cert-manager/templates/NOTES.txt:15]: You should fix it in this PR.

File: stable/cert-manager/templates/00-namespace.yaml (hunk 2 above)
Comment (munnerz) [reply stable/cert-manager/templates/00-namespace.yaml:1]: It is used when generating the 'static manifests' for those that don't want to use helm (https://github.com/jetstack/cert-manager/tree/master/contrib/manifests/cert-manager).

We use `helm template` to generate these files, hence adding it as an option in the Helm chart makes it easy for us to include a namespace resource.

Comment (unguiculus) [reply stable/cert-manager/templates/00-namespace.yaml:1]: Still it doesn't make sense in the Helm chart. Setting it to true will cause an error.

**Pull request merged**